config/config.go

```diff
@@ -42,6 +44,12 @@ const (
 	PaymentChannelStorageTypeKey   = "payment_channel_storage_type"
 	PaymentChannelStorageClientKey = "payment_channel_storage_client"
 	PaymentChannelStorageServerKey = "payment_channel_storage_server"
+	//configs for Daemon Monitoring and Notification
+	AlertsEMail                 = "alerts_email"
+	HeartbeatServiceEndpoint    = "heartbeat_svc_end_point"
+	NotificationServiceEndpoint = "notification_svc_end_point"
+	ServiceHeartbeatType        = "service_heartbeat_type"
```

Comment:
Could you please add documentation for new fields into daemon README.md (https://github.com/singnet/snet-daemon#configuration). Also two new fields from previous PR.

Reply:
Ah, I see that metrics documentation is added to metrics package. Then could you please add the link to it into main configuration secition https://github.com/singnet/snet-daemon#configuration

Reply:
Updated and checked-in